src/promptflow-evals/promptflow/evals/evaluate/_utils.py

```diff
@@ -0,0 +1,9 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import json
+
+
+def load_jsonl(path):
+    with open(path, "r", encoding="utf-8") as f:
```

Comment:
NIT: On windows, by default the file encoding is set to `utf-16`
import chardet

```
# Detect the encoding
with open('your_file.txt', 'rb') as f:
    result = chardet.detect(f.read())

print(result)

# Read the file with the detected encoding
with open('your_file.txt', 'r', encoding=result['encoding']) as f:
    content = f.read()
```

Or we can accept a parameter for encoding and default to `utf-8`?

Reply:
default encoding on windows was causing issues with tqdm progress bar hence set it to utf-8 here. However this method is not used/needed anymore so will remove it.